conans/client/manager.py

```diff
@@ -95,7 +103,8 @@ def package_files(self, reference, package_folder, profile, force):
         remote_proxy = ConanProxy(self._client_cache, self._user_io, self._remote_manager,
                                   remote_name=None, update=False, check_updates=False,
                                   manifest_manager=None)
-        loader = ConanFileLoader(self._runner, self._client_cache.settings, profile)
+
+        loader = ConanFileLoader(self._runner, self._client_cache.settings, self._profile_with_defaults(profile))
```

Comment:
The default is always applied, the same as the old [settings_defaults] in the conan.conf, right?